### PR TITLE
UCSC Tool Cleanup

### PR DESCRIPTION
**Pull request by jmchilton**
- Add citations to used UCSC data source tools
- Remove previously deprecated and likely broken UCSC proxy framework and tools.

### CODE REVIEW

**Comment by bgruening** [comment]
Should we move these tools into https://github.com/galaxyproject/tools-devteam/tree/master/legacy

**Comment by jmchilton** [comment]
Do these tools do anything anymore or do they depend on removed functionality in Galaxy? If they only work with versions of Galaxy that they are distributed with I don't think we need to move them into legacy. If I misunderstand the tools though, they should probably just stay in the distribution.

File: tools/data_source/ucsc_tablebrowser_archaea.xml [hunk 1]
```diff
@@ -39,4 +39,8 @@
         <data name="output" format="tabular" label="${tool.name} on ${organism}: ${table} (#if $description == 'range' then $getVar( 'position', 'unknown position' ) else $description#)"/>
     </outputs>
     <options sanitize="False" refresh="True"/>
+    <citations>
+        <citation type="doi">10.1093/database/bar011</citation>
+        <citation type="doi">10.1101/gr.229102</citation>
+    </citations>
```
Comment (blankenberg) [inline tools/data_source/ucsc_tablebrowser_archaea.xml:46]: Is there an additional separate citation for the archaea browser: http://archaea.ucsc.edu/cite.html

**Comment by blankenberg** [comment]
Well, the /ucsc_proxy/index controller is still in the code and active - but possibly not working. It should probably be removed though...

**Comment by jmchilton** [comment]
Added archaea citation and rebase.

**Comment by bgruening** [comment]
Who knows who is using them and in devteam/legacy they are easier to find than in the git history. Just an idea, I'm fine with removing.

**Comment by jmchilton** [comment]
Okay - @blankenberg you are right it is still in there. I remember after 926c1b1ecf3b2e215551d414f795aeb2a81b4a11 there was a plan to remove it but that didn't happen. Should I just do that as part of this?

**Comment by blankenberg** [comment]
@jmchilton This is a good enough opportunity as any I would think :)

**Comment by jmchilton** [comment]
@blankenberg Done.

**Pull request merged**